Formula/age-plugin-yubikey.rb

```diff
@@ -0,0 +1,33 @@
+class AgePluginYubikey < Formula
+  desc "Plugin for encrypting files with age and PIV tokens such as YubiKeys"
+  homepage "https://github.com/str4d/age-plugin-yubikey"
+  url "https://github.com/str4d/age-plugin-yubikey/archive/v0.3.0.tar.gz"
+  sha256 "3dfd7923dcbd7b02d0bce1135ff4ba55a7860d8986d1b3b2d113d9553f439ba9"
+  license "MIT"
+  head "https://github.com/str4d/age-plugin-yubikey.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  uses_from_macos "expect" => :test
+  uses_from_macos "pcsc-lite"
+
+  on_linux do
+    depends_on "pkg-config" => :build
+  end
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    (testpath/"test.exp").write <<~EOS
+      set timeout 1
+      spawn "#{bin}/age-plugin-yubikey"
+      expect {
+        "Let's get your YubiKey set up for age!" { exit 0 }
```

Comment:
Note that this depends on the `LANG` environment variable (0.3.0 does not contain any translations, but future releases will), so you probably want to set `LANG=en` for this test to pin it.

Reply:
Done!